zmq/_future.py

```diff
@@ -260,24 +241,25 @@ def close(self, linger: int | None = None) -> None:
 
     def get(self, key):
         result = super().get(key)
-        if key == EVENTS:
-            self._schedule_remaining_events(result)
+        # if key == EVENTS:
+        #     self._schedule_remaining_events(result)
```

Comment:
This is one of the things we will need to keep some version of. This is part of handling the edge-triggered FD: all tasks waiting on `wait_readable` must wake if EVENTS is accessed from anywhere.